### ISSUE TITLE
read_dct Qt documentation 

### ISSUE THREAD

**Issue by orique-uoc** [martinbenes1996/jpeglib#20]
Hello,

I'm testing jpeglib for a master thesis. For read_dct, I see we can get luminance, blue-difference chrominance, red-difference chrominance and quantization tables. For a test image, I get these quantization tables:

```
[[[ 6  4  4  6  9 11 12 16]
  [ 4  5  5  6  8 10 12 12]
  [ 4  5  5  6 10 12 12 12]
  [ 6  6  6 11 12 12 12 12]
  [ 9  8 10 12 12 12 12 12]
  [11 10 12 12 12 12 12 12]
  [12 12 12 12 12 12 12 12]
  [16 12 12 12 12 12 12 12]]

 [[ 7  7 13 24 20 20 17 17]
  [ 7 12 16 14 14 12 12 12]
  [13 16 14 14 12 12 12 12]
  [24 14 14 12 12 12 12 12]
  [20 14 12 12 12 12 12 12]
  [20 12 12 12 12 12 12 12]
  [17 12 12 12 12 12 12 12]
  [17 12 12 12 12 12 12 12]]]
```
 
Should I assume qt[0] is the luminance quantization table and qt[1] is the chrominance quantization table?

Thanks!

**Comment by martinbenes1996** [comment]
Hello,

## TL;DR
```python
jpeg = jpeglib.read_dct('file.jpeg')
qt_y = jpeg.qt[jpeg.quant_tbl_no[0]]  # Y QT
qt_cb = jpeg.qt[jpeg.quant_tbl_no[1]]  # Cb QT
qt_cr = jpeg.qt[jpeg.quant_tbl_no[2]]  # Cr QT
```

## Explanation
Usual (sequential) JPEG image can carry up to 4 quantization tables.
The most common for color JPEGs is to see two QTs: one for Y, and one for both Cb and Cr (i.e., shared).
The association between the slot and component can be acquired via `quant_tbl_no` attribute of `DCTJPEG` instance.

 

These quantization tables are associated to the components, and can be shared between t.
